build.sbt

```diff
@@ -1,14 +1,14 @@
 import sbt.Keys._
 
 name := "fezziwig"
-scalaVersion := "2.13.1"
+scalaVersion := "2.13.6"
 crossScalaVersions := Seq("2.12.10", scalaVersion.value)
 organization := "com.gu"
 
-val circeVersion = "0.12.1"
+val circeVersion = "0.14.1"
 
 publishMavenStyle := true
-publishArtifact in Test := false
+Test / publishArtifact := false
```

Comment:
the old "in" syntax was discouraged since 1.0 I believe, and in 1.5 it has been deprecated into a red warning.